src/modules/previewpane/common/handlers/PreviewHandlerBase.cs

```diff
@@ -150,7 +150,7 @@ public void SetTextColor(COLORREF color)
         }
 
         /// <summary>
-        /// Provide instance of the implementation of <see cref="IPreviewHandlerControl"/>. Should be overide by the implementation class with control object to be used.
+        /// Provide instance of the implementation of <see cref="IPreviewHandlerControl"/>. Should be override by the implementation class with control object to be used.

```

Comment:
`Should be overridden by the implementation class with a control object to be used.`

Reply:
```suggestion
        /// Provide instance of the implementation of <see cref="IPreviewHandlerControl"/>. Should be overridden by the implementation class with a control object to be used.
```